src/actions/breakpoints.js

```diff
@@ -36,17 +44,55 @@ async function _getGeneratedLocation(source, sourceMaps, location) {
   return await sourceMaps.getGeneratedLocation(location, source.toJS());
 }
 
+async function syncClientBreakpoint(
+  source,
+  client,
+  sourceMaps,
+  pendingBreakpoint,
+  location
```

Comment:
It's not really clear to me if you need location because it's a composite of many things